client/src/components/pages/create-task/common/create-tabs/create-text/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState } from 'react';
+import { ITabProps } from '../types';
+
+import styles from './styles.module.scss';
+
+export const CreateText: React.FC<ITabProps> = ({ text, onChange, editable }) => {
+	const [value, setValue] = useState<string>(text);
```

Comment:
We don't need local state for tab value here. Value should be recieved from parent container.